python/python/tests/test_transaction.py

```diff
@@ -419,6 +420,12 @@ async def transaction_test(
         alpha=True,
     )
     args.append(4)
+    transaction.sadd(key21, ["one"])
```

Comment:
Can you re-use `key7` instead?  I think they're prefer to avoid adding keys for a single command.